src/safe-ee.js

```diff
@@ -0,0 +1,79 @@
+const EventEmitter = require('events/')
+
+var R = typeof Reflect === 'object' ? Reflect : null
```

Comment:
Is this a polyfill for IE? Might be worth adding a comment about the purpose of this.

Reply:
This is from no events source, so idk